index.html

```diff
@@ -4,9 +4,89 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Library</title>
-    <script type="text/javascript" src="./scripts/library.js"></script> 
+    <link type="text/css" rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.2/css/bulma.min.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
 </head>
 <body>
+  <section class="hero">
+    <div class="hero-body">
+      <div class="container">
+        <h1 class="title">
+          My Library
+        </h1>
+        <h2 class="subtitle">
+          Look at my best books.
+        </h2>
+      </div>
+    </div>
+  </section>
+  <div class="container">
+    <div id="root">
+        <table class="table is-bordered" id="content">
+        </table>
+    </div>
+
+    <a class="button is-primary" id="new-book" onclick="toggleForm()">New Book</a>
     
+    <div class="message is-info hide" id="first-form">
+      <div class="message-header">
+        <p>Add a new Book</p>
+        <button class="delete" aria-label="delete" onclick="toggleForm()"></button>
+      </div>
+      <div class="message-body">
+        <form class="book-form" >
+            <div class="field">
+              <label class="label">Title</label>
+              <div class="control">
+                <input type="text" class="input" placeholder="" id="title">
+                <p class="help is-danger" id="title-danger"></p>
+              </div>
+            </div>
+            <div class="field">
+              <label class="label">Author</label>
+              <div class="control">
+                <input type="text" class="input" placeholder="" id="author">
+                <p class="help is-danger" id="author-danger"></p>
+              </div>
+            </div>
+            <div class="field">
+              <label class="label">Publisher</label>
+              <div class="control">                         
+                <input type="text" class="input" placeholder="" id="publisher">                
+              </div>
+            </div>
+            <div class="field">
+              <label class="label">N° Pages</label>
+              <div class="control">      
+                <input type="text" class="input" placeholder="" id="number-of-pages">                
```

Comment:
consider making this a number [OPTIONAL]
As it stands, I can type anything into here, including non-numeric characters